src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/code/InstalledCode.java

```diff
@@ -127,10 +146,10 @@ public byte[] getCode() {
     }
 
     /**
-     * Equivalent to calling {@link #invalidate(boolean)} with a {@code true} argument.
+     * Equivalent to calling {@link #invalidate(boolean, int)} with {@code true} and {@code 0} as arguments.
      */
     public void invalidate() {
-        invalidate(true);
+        invalidate(true, 0);
```

Comment:
This assigns `ChangeReason::C1_codepatch` to JVMCI invalidations which does not seem right.

Reply:
I believe zero is mapped to `UNKNOWN` in the ChangeReason struct, if I'm not very mistaken here?!

Reply:
Sorry, I was looking at nmethod.hpp in my local source without the changes in this PR. That said, it should probably map to `ChangeReason::JVMCI_invalidate_nmethod` right?
BTW, seems like `ChangeReason::JVMCI_invalidate_nmethod_mirror` is unused and could be deleted.